docs/design/jobflow/README.md

```diff
@@ -48,7 +48,13 @@ jobflow job submission complete process：
 
 6. After assigning nodes to Pods, kubelet will get the Pod's configuration from kube-apiserver and start the corresponding containers.
 
+**update jobflow**:
 
+Currently, jobflow does not support the update operation, and the update of jobflow will be blocked through webhook.
+
+**delete jobflow**:
+
+After deleting jobflow, all vcjobs created by jobflow will be deleted directly, no matter what state the vcjobs are in.
```

Comment:
It is recommended to use webhook interception here. If there is a running job, it is not allowed to delete the jobflow. Unless the --force parameter is used when removing

Reply:
Thank you, I will add it in the webhook. When you need to delete the jobflow in the running state, you can only delete it with the command kubectl delete vcjob {vcjobName} --force